engine/src/main/java/org/hibernate/validator/messageinterpolation/HibernateMessageInterpolatorContext.java

```diff
@@ -40,4 +41,9 @@
 	 * @since 5.4.1
 	 */
 	Map<String, Object> getExpressionVariables();
+
+	/**
+	 * @return the property path to the value from {@code rootBean}
```

Comment:
Maybe `@return the property path to the validated constraint starting from {@code rootBean}` ?

Reply:
fixed